Yoast/wordpress-seo
# Add reindex interface

## Summary

This PR can be summarized in the following changelog entry:
* Added interface for indexing links.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Always, always do a `grunt build:js` to be sure if everything is up-2-date
* Have some internal links
* Run the reindexation
* See everyting has been reindexed.

Fixes #7280
